src/libaktualizr-isotp/isotpsecondary.cc

```diff
@@ -137,6 +137,12 @@ bool IsoTpSecondary::putMetadata(const RawMetaPack& meta_pack) {
   return conn.Send(out);
 }
 
+data::ResultCode::Numeric install(const Target& target) {
+  (void)target;
+  // TODO: implement an image sending data by using ImageReader and IsoTpSecondary::sendFirmware
+  return data::ResultCode::Numeric::kOk;
+}
+
```

Comment:
Historically this was just reusing the default string sending, right? How hard is it to just replicate that here? I know we don't have a good way to test that at the moment.

Reply:
just implemented it.